### ISSUE TITLE
use consistent "autoIdle" option everywhere (instead of auto-idle on Env)

### ISSUE THREAD

**Issue by tobybellwood** [uselagoon/lagoon-cli#351]
The only place that auto-idle is used as an option is in the Environment command
https://github.com/uselagoon/lagoon-cli/blob/146b2723dce8cab579d66f093c43a9f285a312fa/cmd/environment.go#L164

These should all be the same (ie autoIdle) IMO

**Comment by shreddedbacon** [comment]
I'm really against using camel case for flags, from a UX perspective having to switch case when typing out flags is annoying.
If we're going to be the same across commands, then they should all be `auto-idle` or `autoidle`.

**Comment by tobybellwood** [comment]
:smile: Just pointing out the discrepancy - happy for it to be kebab-case instead

Most of the project subcommands are still camelCase - should we alias them to kebab to avoid b/c issues?
https://uselagoon.github.io/lagoon-cli/commands/lagoon_add_project/
https://uselagoon.github.io/lagoon-cli/commands/lagoon_update_project/

**Comment by shreddedbacon** [comment]
> Most of the project subcommands are still camelCase - should we alias them to kebab to avoid b/c issues? [uselagoon.github.io/lagoon-cli/commands/lagoon_add_project](https://uselagoon.github.io/lagoon-cli/commands/lagoon_add_project/) [uselagoon.github.io/lagoon-cli/commands/lagoon_update_project](https://uselagoon.github.io/lagoon-cli/commands/lagoon_update_project/)

The CLI is still not a v1 so we have some give in breaking change introduction.

**Comment by tobybellwood** [comment]
ugh - it gets a bit messy though with dual strings in AddProject - so the `requiredInputCheck` may need rewriting to support either git-url or gitUrl as an option - or we just deprecate and don't support dual flags.

Also, it looks like we just unmarshal the flags into the patch in updateProject, which isn't going to work if the flags don't match the GraphQL

**Comment by shreddedbacon** [comment]
> ugh - it gets a bit messy though with dual strings in AddProject - so the `requiredInputCheck` may need rewriting to support either git-url or gitUrl as an option - or we just deprecate and don't support dual flags.

Yeah, the `requiredInputCheck` function generally I don't think is actually required either, we have `MarkFlagRequired` in cobra that can do the this. But it still suffers from the problem of double flags.
```
subCmd.MarkFlagRequired("bar")
```

> Also, it looks like we just unmarshal the flags into the patch in updateProject, which isn't going to work if the flags don't match the GraphQL

#321 fixes parts of that issue by moving to the newer machinery library instead of the legacy cli structure.

---
This is a solveable problem, and if we have to make some hacks in the `requiredInputCheck` to evaluate double flags for now then that is fine. Otherwise we take a breaking hit and people will have to adjust
